shaazk/Broadcaster
# #169937366 fix endpoint errors

#### What does this PR do?
- Fix endpoint errors .
#### Description of Task to be completed?
- A user will be able to all only his or her readflags or interventions 
#### Any background context you want to provide?
- N/A
#### How should it be tested?
 - clone BroadCaster on your computer and open the older in your VS Code or any other HTML IDE. in your terminal, run nodemon by typing npm start and then open your postman and run http://localhost:4000/api/v1/red-flags/id under GET function and in body, insert credentials and click on send
- in your VSCode after running nodemon, you can click on Ctrl + the link in the terminal where you see the server running on http://localhost:4000/api/v1/red-flags/id 
NB: use different tokens to create incidents and view them, you will be able to only see the redflags or interventions created in only at the account you are in
#### What are the relevant pivotal tracker stories?
- [#169937366](https://www.pivotaltracker.com/story/show/169937366)
